assets/js/components/notifications/UserInputSettings.js

```diff
@@ -54,6 +54,7 @@ export default function UserInputSettings() {
 			ctaLabel={ __( 'Let’s go', 'google-site-kit' ) }
 			dismiss={ __( 'Remind me later', 'google-site-kit' ) }
 			winImage={ global._googlesitekitLegacyData.admin.assetsRoot + personSitImage }
+			className="googlesitekit-user-input__notification"
```

Comment:
Let's move this up next to the `id` as we usually have `className` as one of the first props.

Reply:
Updated @aaemnnosttv

Reply:
This is still at the bottom here 🙂